jsconfig.json

```
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src"
```

You might unlock more power with "." :smirk: 

```suggestion
    "baseUrl": "."
```